### ISSUE TITLE
TestValidMsg_StoreDoc crashes with nil pointer dereference

### ISSUE THREAD

**Issue by gsora** [commercionetwork/commercionetwork#61]
## Summary of Bug

`TestValidMsg_StoreDoc` crashes when testing, with a nil pointer dereference.

## Version

`69d4926a07529192555ae7036e1a882d171e4f8c`

## Steps to Reproduce

1. Execute `TestValidMsg_StoreDoc`

---

This is the stacktrace produced by the crash:

```
--- FAIL: TestValidMsg_StoreDoc (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xd9c7ff]

goroutine 20 [running]:
testing.tRunner.func1(0xc00017f900)
	/usr/lib/go/src/testing/testing.go:874 +0x3a3
panic(0xeb0540, 0x1a5ad20)
	/usr/lib/go/src/runtime/panic.go:679 +0x1b2
github.com/cosmos/cosmos-sdk/x/supply/internal/keeper.Keeper.SetModuleAccount(...)
	/home/gsora/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.34.4-0.20191013030331-92ea174ea6e6/x/supply/internal/keeper/account.go:60
github.com/commercionetwork/commercionetwork/x/id/internal/keeper.SetupTestInput(0x36c91b702d18d2a, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/gsora/commercio/commercionetwork/x/id/internal/keeper/test_commons.go:75 +0x15bf
github.com/commercionetwork/commercionetwork/x/id/internal/keeper.TestValidMsg_StoreDoc(0xc00017f900)
	/home/gsora/commercio/commercionetwork/x/id/internal/keeper/handler_test.go:15 +0x74
testing.tRunner(0xc00017f900, 0x11e00a0)
	/usr/lib/go/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/usr/lib/go/src/testing/testing.go:960 +0x350
FAIL	github.com/commercionetwork/commercionetwork/x/id/internal/keeper	0.097s
FAIL
```

**Comment by gsora** [comment]
The crash occurs when `SetModuleAccount` gets called.

`SetModuleAccount` is just an alias for `k.ak.SetAccount(ctx, macc)`, where `k` is the Keeper method variable: the `ak` struct field is initialized to nil, hence the nil pointer dereference.

---

After some digging, looks like something is wrong with the `NewKeeper` function:

```go
// NewKeeper creates new instances of the CommercioID Keeper
func NewKeeper(cdc *codec.Codec, storeKey sdk.StoreKey, accKeeper auth.AccountKeeper, supplyKeeper supply.Keeper) Keeper {
        // ...code here...
	return Keeper{
		storeKey:  storeKey,
		cdc:       cdc,
		accKeeper: accKeeper,
	}
}
```

By initializing the `Keeper.supplyKeeper` field with the `supplyKeeper` parameter, the test works.

**Comment by RiccardoM** [comment]
Closes with #64
